src/engine/Engine.h

```diff
@@ -279,7 +279,7 @@ class Engine : public IEngine, public SignalHandler::Signalable
     /*
       The existing transcendental constraints.
     */
-    List<TranscendentalConstraint *> _tsConstraints;
+    List<NonlinearConstraint *> _tsConstraints;
```

Comment:
_tsConstraints should probably be renamed, too

Reply:
done